spec/providers/coreIntegration/udpsocket.integration.src.js

```diff
@@ -8,71 +8,70 @@ module.exports = function(provider, setup) {
   });
 
   it("Connects, has state, and sends/receives data", function (done) {
-    // Currently a copy of 'receives data' unit test from firefox, change it
+    const todo = [];  // Pending tasks to complete before calling |done()|.
```

Comment:
As you may have noticed, this test used `const` exclusively.  Would you like me to change them all?